packages/icons/src/library/direction-nw.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const directionNW = (
```

Comment:
I think these should be named `arrowTopLeft` and such for consistency with existing icons.